src/deploy-cloudrun.ts

```diff
@@ -19,10 +19,26 @@ import { getExecOutput } from '@actions/exec';
 import * as toolCache from '@actions/tool-cache';
 import * as setupGcloud from '@google-github-actions/setup-cloud-sdk';
 import path from 'path';
+import { parseDeployResponse, parseUpdateTrafficResponse } from './output-parser';
 
 export const GCLOUD_METRICS_ENV_VAR = 'CLOUDSDK_METRICS_ENVIRONMENT';
 export const GCLOUD_METRICS_LABEL = 'github-actions-deploy-cloudrun';
 
+/**
+ * DeployCloudRunOutputs are the common GitHub action outputs created by this action
+ */
+export interface DeployCloudRunOutputs {
+  url?: string | null | undefined;
```

Comment:
I think by using `url?: string`, you implicitly allow undefined too. So maybe shorten this to:

```
url?: string
```